### ISSUE TITLE
Quick Start v2 VoiceOver: close button is not accessible

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#11491]
Steps to reproduce:

1. Log out of the WPiOS app.
1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. Go to WPiOS and double tap the "Sign up for WordPress.com" button.
1. Follow the prompts until you reach "Want a little help getting started?"
1. Double tap the "Yes, Help Me" button.
1. Double tap the close button.

Result: the close button is not labeled so tapping it with VoiceOver on just says "button". ([9s](https://cloudup.com/chM5_zopDkr))

<img src="https://user-images.githubusercontent.com/1119271/56327182-80e30280-6136-11e9-8d54-77d7872408b6.png" width="270" alt="quick-start-v2-close-button">
<p style="clear:both;"><sup>Tested with WP Internal 12.2.0.20190412 on iPhone 6S iOS 12.2.</sup></p>

<p style="clear:both;"><sub>(internal reference: p77Llu-bQ6-p2 h/t @etoledom)</sub></p>
